### PR TITLE
Paginated list of chains

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #1537

Changes proposed in this pull request:
- Load pages of chain info
- Paginated list of chain items
- Pass selected chain through add safe flow
- Adjust nav graph

@gnosis/mobile-devs

### CODE REVIEW

File: data/src/main/java/io/gnosis/data/models/Chain.kt [hunk 1]
```diff
@@ -19,7 +20,7 @@ data class Chain(
 
     @ColumnInfo(name = COL_BACKGROUND_COLOR)
     val backgroundColor: String
-) {
+) : Serializable {
```
Comment (biafra23) [inline data/src/main/java/io/gnosis/data/models/Chain.kt:23]: Why is this needed?

**Review by biafra23 — APPROVED** [review]

File: data/src/main/java/io/gnosis/data/models/Chain.kt (hunk 1 above)
Comment (elgatovital) [reply data/src/main/java/io/gnosis/data/models/Chain.kt:23]: to pass it as an argument in a bundle
